src/pages/namjanghyun/Login/Login.js

```diff
@@ -1,7 +1,112 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./Login.scss";
 
 function LoginNam() {
-  return <h1>Hello world</h1>;
+  const navigate = useNavigate(); // 이동
+
+  const [inputValue, setInputValue] = useState({ email: "", password: "" });
+
+  const handleLogin = (event) => {
+    const { name, value } = event.target;
+    setInputValue({ ...inputValue, [name]: value });
+  };
+
+  const handleDisabled = () => {
+    return inputValue.email.indexOf("@") >= 0 && inputValue.password.length >= 5
+      ? ""
+      : "disabled";
+  };
```

Comment:
- 1차적으로, 일반 변수로 만들어버릴 수 있음
```suggestion
  const isInputValid =  inputValue.email.indexOf("@") >= 0 && inputValue.password.length >= 5
```
- disabled 속성이 받는 값의 형태는 boolean!